src/bundles/communication/GlobalStateController.ts

```diff
@@ -0,0 +1,117 @@
+import { type MultiUserController } from './MultiUserController';
+
+/**
+ * Controller for maintaining a global state across all devices.
+ * Depends on MQTT implementation in MultiUserController.
+ *
+ * @param topicHeader Identifier for all global state messages, must not include '/'.
+ * @param multiUser Instance of multi user controller.
+ * @param callback Callback called when the global state changes.
+ */
+export class GlobalStateController {
+  private topicHeader: string;
+  private multiUser: MultiUserController;
+  private callback: (state: any) => void;
+  globalState: any;
```

Comment:
Is there any way to avoid `any`? Likewise, in the callback passed to the `initGlobalState` function in `Communications.ts`

Reply:
Yes, 'any' is quite justified here. The state can literally be **anything**.